lib/features/email/data/network/email_api.dart

```diff
@@ -90,6 +96,10 @@ class EmailAPI with HandleSetErrorMixin {
       ..addIds({emailId.id})
       ..addProperties(ThreadConstants.propertiesGetEmailContent)
       ..addFetchHTMLBodyValues(true);
+    if (withIdentityHeader) {
+      getEmailMethod.addProperties(Properties({
+        IndividualHeaderIdentifier.identityHeader.value}));
+    }
```

Comment:
IMO, we should add `Properties` as a parameter of the `getEmailContent` function so that more `properties` can be used later.

Reply:
Done